extensions/m3o/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "m3o",
+  "title": "M3O",
+  "description": "Explore, discover and consume public APIs as simpler programmable building blocks.",
+  "icon": "command-icon.png",
+  "author": "andreaselia",
+  "categories": [
+    "Data",
+    "Web"
+  ],
+  "license": "MIT",
+  "commands": [{
+    "name": "avatar",
+    "title": "Avatar",
```

Comment:
As per https://developers.raycast.com/basics/prepare-an-extension-for-store#extensions-and-commands-naming, command title should have a verb. Let's use `Generate an Avatar` here

Reply:
My bad, I missed that. This has been resolved.